projects/storefrontlib/shared/components/chat-messaging/avatar/avatar.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { MessageEvent } from '../messaging';
+
+@Component({
+  selector: 'cx-avatar',
+  templateUrl: './avatar.component.html',
+})
+export class AvatarComponent implements OnInit {
+  @Input() event: MessageEvent;
+  iconTypes = ICON_TYPE;
+
+  constructor() {}
+
+  ngOnInit(): void {}
+
+  getIntitials(name: string) {
```

Comment:
It could be a name, fullname or email ( depends on the hosting component )
+ typo "Initials"
```suggestion
  getInitials(author: string) {
```